test/autoproj/daemon/test_git_poller.rb

```diff
@@ -96,6 +96,17 @@ def git_url(owner, name)
                 "git@github.com:#{owner}/#{name}.git"
             end
 
+            describe "#parse_buildconf_branch" do
+                it "parses a github URL" do
+                    parsed = @poller.parse_buildconf_branch(
+                        "autoproj/wetpaint/tidewise/drivers-something/pulls/22"
```

Comment:
This is no longer a valid branch name (see [this](https://github.com/rock-core/autoproj-daemon/blob/3b8e23d9837e93469a09d5169ad63c830538fc09/lib/autoproj/daemon/git_poller.rb#L252) and [this](https://github.com/rock-core/autoproj-daemon/blob/3b8e23d9837e93469a09d5169ad63c830538fc09/lib/autoproj/daemon/git_api/url.rb#L55)).

The rationale being that since we now support both gitlab and github repos in the same workspace, the "local" path (formerly `#{owner}/#{name}`) is not enough to resolve branches to packages (and repositories). Hence the `#full_path`.

Reply:
Oh ... got that now. Just modified the commit to add an explicit test.